packages/dx-react-scheduler-material-ui/src/templates/views/common/table.jsx

```diff
@@ -1,17 +1,23 @@
 import * as React from 'react';
+import { styled } from '@mui/material/styles';
 import * as PropTypes from 'prop-types';
 import classNames from 'clsx';
 import TableMUI from '@mui/material/Table';
 import TableBody from '@mui/material/TableBody';
-import makeStyles from '@mui/styles/makeStyles';
 import { getWidthInPixels } from '../../utils';
 import {
   CELL_WIDTH, SMALL_CELL_WIDTH,
   XS_CELL_WIDTH, SMALL_LAYOUT_MEDIA_QUERY, LAYOUT_MEDIA_QUERY,
 } from '../../constants';
 
-const useStyles = makeStyles({
-  table: {
+const PREFIX = 'Table';
+
+const classes = {
+  table: `${PREFIX}-table`,
+};
+
+const StyledTableMUI = styled(TableMUI)({
+  [`&.${classes.table}`]: {
     tableLayout: 'fixed',
     minWidth: cellsNumber => getWidthInPixels(cellsNumber, CELL_WIDTH),
```

Comment:
Is it working? in the line 25 and 28 the same question

Reply:
Didn't find any problem with this

Reply:
It is a function, and it is not called